doc_src/en/OmegaT5_ProjectFolder.xml

```diff
@@ -4,143 +4,159 @@
+	  <listitem>
+		<para>use a reference translation memory folder that you have set up for
+		related projects, and so on.</para>
+	  </listitem>
+	</itemizedlist>
+	
+	<para>Creating a project gives it a core hierarchy, but its final structure is
+	not fixed. You can always remove or add files and folders to the
+	project.</para>
+
+	<note>
+	  <para>You can also create OmegaT projects either manually or with scripts by
+	  copying a set of prewritten files to a new folder:</para>
+	  
+	  <itemizedlist>
+		<listitem>
+		  <para>Any given folder that contains a valid
+		  <filename>omegat.project</filename> file will be recognized as a
+		  translation project by OmegaT (even if it later needs manual
+		  adjustments).</para>
+		</listitem>
+
+		<listitem>
+		  <para>If the folder is an empty repository and the
+		  <filename>omegat.project</filename> file contains remote repository
+		  information, the project will be recognized as a team project by
+		  OmegaT:</para>
+		  
+		  <programlisting>...
     &lt;external_command&gt;&lt;/external_command&gt;
     &lt;repositories&gt;
         &lt;repository type="git" url="https://URL/OF/THE/REMOTE/PROJECT/REPOSITORY"&gt;
             &lt;mapping local="/" repository="/"/&gt;
         &lt;/repository&gt;
     &lt;/repositories&gt;
-&lt;/project&gt;</programlisting>
+&lt;/project&gt;
+		  </programlisting>
 
-		<para>See the <link linkend="how.to.setup.team.project"
-		endterm="how.to.setup.team.project.title"/> how-to for details.</para>
-	  </listitem>
-	</itemizedlist>
-  </note>
+		  <para>See the <link linkend="how.to.setup.team.project"
+		  endterm="how.to.setup.team.project.title"/> how-to for details.</para>
+		</listitem>
+	  </itemizedlist>
+	</note>
+  </section>
+
+  <section>
+	<title>Minimal contents</title>
+
+	<para>A project is a folder containing, at a minimum, the following:</para>
+	
+	<variablelist>
+	  <varlistentry id="project.folder.omegat">
+		<term id="project.folder.omegat.title">omegat</term>
+		<listitem>
+		  <para>This is the project folder that always contains <link
+		  linkend="project.folder.project.save.tmx"
+		  endterm="project.folder.project.save.tmx.title"/>, the project
+		  translation memory, and <link linkend="project.folder.project.stats"
+		  endterm="project.folder.project.stats.title"/>, the statistics data file
+		  for the project.</para>
+
+		  <para>Other files will be added over the course of the translation.</para>
+
+		  <para>See <link linkend="project.folder.omegat.folder">below</link> for
+		  details.</para>
+		</listitem>
+	  </varlistentry>
+
+	  <varlistentry id="project.folder.omegat.project.file">
+		<term id="project.folder.omegat.project.file.title">omegat.project</term>
+		<listitem>
+		  <para>This file contains the project parameters defined in the <link
+		  linkend="dialogs.project.properties">project properties</link>: source and
+		  target languages, their respective tokenizers, the type of segmentation
+		  used, etc.</para>
```

Comment:
I would change "...project properties: source and target languages, their respective tokenizers, the type of segmentation used, etc."

to

"... project properties, such as the source and target languages, their respective tokenizers and the type of segmentation."

Everything else looks good!